### PR TITLE
change redis tls cert files name to valkey tls cert files

### PR DESCRIPTION
**Pull request by Shivshankar-Reddy**
This PR covers changing the redis.crt and redis.key to valkey certs for tls testing.

here are the files generated by the gen-test-certs.sh script.

```
valkey$ ls tests/tls/
ca.crt  ca.key  client.crt  client.key  openssl.cnf  server.crt  server.key  valkey.crt  valkey.dh  valkey.key
```

runtest results for `./runtest --tls`
```

/valkey$ ./runtest --tls
Cleanup: may take some time... OK
Starting test server at port 21079
[ready]: 2423683
Testing unit/printver
[ready]: 2423684
Testing unit/dump
[ready]: 2423685
Testing unit/auth
[ready]: 2423686
Testing unit/protocol
[ready]: 2423687
Testing unit/keyspace
[ready]: 2423688
Testing unit/scan
[ready]: 2423689
Testing unit/info
[ready]: 2423690
Testing unit/info-command
[ready]: 2423691
Testing unit/type/string
[ready]: 2423692
Testing unit/type/incr
[ready]: 2423693
Testing unit/type/list
[ready]: 2423694
Testing unit/type/list-2
[ready]: 2423695
Testing unit/type/list-3
[ready]: 2423696
Testing unit/type/set
[ready]: 2423697
Testing unit/type/zset
[ready]: 2423698
Testing unit/type/hash
[ok]: DEL against a single item (2 ms)
-------
-------
  5 seconds - unit/cluster/slot-ownership
  8 seconds - unit/cluster/human-announced-nodename
  12 seconds - unit/cluster/hostnames
  42 seconds - unit/scripting
  125 seconds - unit/dump
  12 seconds - unit/cluster/failure-marking
  32 seconds - unit/wait
  28 seconds - unit/client-eviction
  44 seconds - unit/obuf-limits
  20 seconds - unit/cluster/cluster-response-tls
  20 seconds - unit/cluster/links
  61 seconds - unit/introspection
  58 seconds - unit/geo
  78 seconds - unit/hyperloglog
  111 seconds - unit/maxmemory
  189 seconds - integration/replication
  184 seconds - integration/replication-psync
  0 seconds - list-large-memory
  1 seconds - set-large-memory
  0 seconds - bitops-large-memory
  2 seconds - violations
  2 seconds - defrag

\o/ All tests passed without errors!

Cleanup: may take some time... OK

```
Other runtest such as runtest-cluster runtest-moduleapi and runtest-sentinel are also passed.

### CODE REVIEW

File: TLS.md [hunk 1]
```diff
@@ -38,22 +38,22 @@ invoked so sample certificates/keys are available):
 
 For TLS built-in mode:
     ./src/redis-server --tls-port 6379 --port 0 \
```
Comment (hwware) [inline TLS.md]:
```suggestion
    ./src/valkey-server --tls-port 6379 --port 0 \
```

File: TLS.md [hunk 2]
```diff
@@ -38,22 +38,22 @@ invoked so sample certificates/keys are available):
...
-        --tls-cert-file ./tests/tls/redis.crt \
-        --tls-key-file ./tests/tls/redis.key \
+        --tls-cert-file ./tests/tls/valkey.crt \
+        --tls-key-file ./tests/tls/valkey.key \
         --tls-ca-cert-file ./tests/tls/ca.crt
 
 For TLS module mode:
     ./src/redis-server --tls-port 6379 --port 0 \
```
Comment (hwware) [inline TLS.md]:
```suggestion
    ./src/valkey-server --tls-port 6379 --port 0 \
```

File: TLS.md [hunk 3]
```diff
@@ -38,22 +38,22 @@ invoked so sample certificates/keys are available):
...
-        --tls-cert-file ./tests/tls/redis.crt \
-        --tls-key-file ./tests/tls/redis.key \
+        --tls-cert-file ./tests/tls/valkey.crt \
+        --tls-key-file ./tests/tls/valkey.key \
         --tls-ca-cert-file ./tests/tls/ca.crt \
         --loadmodule src/redis-tls.so
 
 To connect to this Redis server with `redis-cli`:
```
Comment (hwware) [inline TLS.md]:
```suggestion
To connect to this Valkey server with `valkey-cli`:
```

File: TLS.md [hunk 4]
```diff
@@ -38,22 +38,22 @@ invoked so sample certificates/keys are available):
...
 
     ./src/redis-cli --tls \
```
Comment (hwware) [inline TLS.md]:
```suggestion
    ./src/valkey-cli --tls \
```

File: redis.conf [hunk 5]
```diff
@@ -228,7 +228,7 @@ tcp-keepalive 300
 # required by older versions of OpenSSL (<3.0). Newer versions do not require
 # this configuration and recommend against it.
 #
-# tls-dh-params-file redis.dh
+# tls-dh-params-file valkey.dh
 
 # Configure a CA certificate(s) bundle or directory to authenticate TLS/SSL
 # clients and peers.  Redis requires an explicit configuration of at least one
```
Comment (hwware) [inline redis.conf]:
```suggestion
# clients and peers. Valkey requires an explicit configuration of at least one
```

File: tests/unit/tls.tcl [hunk 6]
```diff
@@ -100,8 +100,8 @@ start_server {tags {"tls"}} {
             set master_port [srv 0 port]
 
             # Use a non-restricted client/server cert for the replica
-            set redis_crt [format "%s/tests/tls/redis.crt" [pwd]]
```
Comment (hwware) [inline tests/unit/tls.tcl:103]: pls update redis_crt to valkey_crt

File: tests/unit/tls.tcl [hunk 7]
```diff
@@ -100,8 +100,8 @@ start_server {tags {"tls"}} {
...
-            set redis_key [format "%s/tests/tls/redis.key" [pwd]]
```
Comment (hwware) [inline tests/unit/tls.tcl:104]: pls update redis_key to valkey_key

File: tests/unit/tls.tcl [hunk 8]
```diff
@@ -100,8 +100,8 @@ start_server {tags {"tls"}} {
...
+            set redis_crt [format "%s/tests/tls/valkey.crt" [pwd]]
+            set redis_key [format "%s/tests/tls/valkey.key" [pwd]]
 
             start_server [list overrides [list tls-cert-file $redis_crt tls-key-file $redis_key] \
```
Comment (hwware) [inline tests/unit/tls.tcl]:
```suggestion
            start_server [list overrides [list tls-cert-file $valkey_crt tls-key-file $valkey_key] \
```

File: utils/gen-test-certs.sh [hunk 9]
```diff
@@ -6,7 +6,7 @@
 #   tests/tls/redis.{crt,key}       A certificate with no key usage/policy restrictions.
```
Comment (hwware) [inline utils/gen-test-certs.sh]:
```suggestion
#   tests/tls/valkey.{crt,key}       A certificate with no key usage/policy restrictions.
```

**Comment by Shivshankar-Reddy** [comment]
Due to cyclic conflicts unable resolve manually so abondoning this PR

**Comment by Shivshankar-Reddy** [comment]
created https://github.com/valkey-io/valkey/pull/186

**Pull request closed without merging**